connectivity/suite.go

```diff
@@ -38,86 +39,120 @@ var (
 	//go:embed manifests/client-egress-to-fqdns-google.yaml
 	clientEgressToFQDNsGooglePolicyYAML string
 
-	//go:embed manifests/echo-ingress-from-other-client.yaml
-	echoIngressFromOtherClientPolicyYAML string
```

Comment:
same here. Having `other` in the name allows for easy run of only this test with `--test other`...